### PR TITLE
Add .gitignore

### PR DESCRIPTION
**Pull request by blueberry1312**
### Milestone 1:

- Added .gitignore.
- Added linters.
- Update README.md.
- Added index.html of mobile version.
- Added style.css and images.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <body>
+        <header>
+            <div class="toolbar">
+                <a href="" class="logo-placeholder">
+                    <h3>Welcome</h3>
+                </a>
+                <button type="button" class="menu">
+                    <img src="assets/img/menu.svg" alt="menu link">
+                </button>
+            </div>
```
Comment (franclobo) [inline index.html]: - [x] Kindly close the `header` tag after the hamburger button and please 🙏🏽 put the hamburger button inside a `nav` tag for the best practices of HTML&CSS.

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #227452;
+}
```
Comment (franclobo) [inline assets/css/style.css]: - [x] Kindly delete the extra space between the title and subtitle to look similar to the Figma template. 💡 **Hint:** use margin-bottom and margin-top in the corresponding element.

| Figma template | Your design |
|---|---|
| image | image |

File: assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  align-items: center;
+  text-align: center;
+}
+
+.contact {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  text-align: center;
+  list-style: none;
```
Comment (franclobo) [inline assets/css/style.css]: - [x] Please 🙏🏽 center the social icons to look similar to the Figma template. **Hint:** use padding.

File: assets/css/style.css [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  font-size: 20px;
+  line-height: 28px;
+  margin: 2em 24px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.headline {
+  background-color: #fff;
+  background-image: url("../../assets/img/background-image-mobile.svg");
```
Comment (franclobo) [inline assets/css/style.css:37]: - [x] Please 🙏🏽 correct the position of the background image to look similar to the Figma template.

| Figma template | Your design |
|---|---|
| image | image |

File: assets/css/style.css [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+  background: none;
+  border: 10px;
+}
+
+.headline button {
+  display: flex;
+  flex-direction: row;
+  background: none;
+  align-items: flex-start;
+  padding: 8px;
```
Comment (franclobo) [inline assets/css/style.css:78]: - [x] I think there is more distance between the social icons and the button, please 🙏🏽 correct the distance to look similar to the Figma template.

| Figma template | Your template |
|---|---|
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @blueberry1312, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/css/style.css (hunk 3 above)
Comment (blueberry1312) [reply assets/css/style.css:87]: Hello @franclobo, thanks for the help, I think I just need to fix the background, could you guide me a bit please? Thank you!

File: assets/css/style.css (hunk 4 above)
Comment (blueberry1312) [reply assets/css/style.css:37]: Hello @franclobo, thanks for the help, I think I just need to fix the background, could you guide me a bit please? Thank you!

**Review by roshan-bajgain — APPROVED** [review]
Hi @blueberry1312, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**